src/server/ApiRoutes/User.js

```diff
@@ -12,4 +13,13 @@ function requireUser(req, res) {
   return true;
 }
 
-export default { requireUser };
+function eligibility(req, res) {
+  const id = req.patronTokenResponse.decodedPatron.sub;
```

Comment:
You should add checks to make sure `patronTokenResponse`, `decodedPatron`, and `sub` are available in the object.

Reply:
OK, but not really sure what to do if they aren't

Reply:
Maybe `eligibility` should be false since there won't be a patron anyway?